api/v1beta1/elfmachine_types.go

```diff
@@ -167,6 +167,12 @@ type ElfMachineStatus struct {
 	// modified by users.
 	// +optional
 	HostServerName string `json:"hostServerName,omitempty"`
+
+	// placementGroupRef is used to lookup the VM placement group.
```

Comment:
// PlacementGroupRef is the reference to the Tower VM Placement Group which this ElfMachine belongs to.